chapters/extensions/VK_KHR_shader_subgroup_uniform_control_flow.md

```diff
@@ -0,0 +1,123 @@
+# VK_KHR_shader_subgroup_uniform_control_flow
+
+## Overview
+
+VK_KHR_shader_subgroup_uniform_control provides stronger guarantees for
+reconvergence of invocations in a shader. If the extension is supported,
+shaders can be modified to include a new attribute that provides the stronger
+guarantees (see GL_EXT_subgroup_uniform_control_flow). This attribute can only
+be applied to shader stages that support subgroup operations (check the
+subgroupSupportedStages feature in the VkPhysicalDeviceVulkan11Properties
```

Comment:
Afaik `VkPhysicalDeviceVulkan11Properties` requires Vulkan 1.2, but the control flow extension is written against Vulkan 1.1. for which `VkPhysicalDeviceSubgroupProperties` can be used to query the supported stages. Not sure if that's too much detail though.

Reply:
Could just say:

Check `VkPhysicalDeviceSubgroupProperties::supportedStages` or `VkPhysicalDeviceVulkan11Properties::subgroupSupportedStages`